bindings/python/src/converters.cpp

```diff
@@ -214,6 +263,8 @@ void bind_converters()
     to_python_converter<std::vector<lt::udp::endpoint>, vector_to_list<lt::udp::endpoint>>();
     to_python_converter<std::vector<std::pair<std::string, int>>, vector_to_list<std::pair<std::string, int>>>();
 
+    to_python_converter<std::map<lt::file_index_t, std::string>, map_to_dict<lt::file_index_t, std::string>>();
```

Comment:
not anymore for python->C++ after I figured out that rename_files was just a setter. But c++->python is used for rename_files